modules/ROOT/Edits/endpoint-sftp-send.adoc

```diff
@@ -0,0 +1,66 @@
+= SFTP Send Endpoint Settings
```

Comment:
ditto edits to "FTP Send Endpoint Settings"; make wordings parallel if not exactly the same.